var/spack/repos/builtin/packages/supernova/package.py

```diff
@@ -1,45 +0,0 @@
-# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
This still needs fixed - we don't want this change to delete the file from the repo